src/components/DocumentList.tsx

```
@@ -57,24 +63,24 @@ const DocumentList: React.FC<DocumentListProps> = ({ documents }) => {
     onOpen: onImageOpen,
     onClose: onImageClose,
   } = useDisclosure();
-
   const toast = useToast();
 
   const [selectedDocument, setSelectedDocument] = useState<any | null>(null);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)